src/main/java/org/jabref/gui/preferences/PreferencesDialog.java

```diff
@@ -127,6 +128,13 @@ private void construct() {
                                                 .withText(PrefsTab::getTabName)
                                                 .install(tabsList);
 
+        TextField searchBox = new TextField();
+        searchBox.setPromptText(Localization.lang("Search"));
+
+        PreferencesSearchHandler searchHandler = new PreferencesSearchHandler(preferenceTabs);
+        tabsList.itemsProperty().bindBidirectional(searchHandler.filteredPreferenceTabsProperty());
+        searchHandler.searchTextProperty().bindBidirectional(searchBox.textProperty());
```

Comment:
Another try would be to pass the `searchBox.textProperty()` as the constructor argument and then add the listener directly (I think you had something similar before). I think both approaches are valid although it's strange that the current code does not work for you.

Reply:
Indeed, that was the previous approach. 
How about another approach where we add a listener on `searchBox`, which makes call to `searchHandler.filterTabs` directly, without the need of having `StringProperty` in between?

Reply:
Yes, that sounds like the easiest solution. Please go ahead and make the corresponding changes. Thanks!

Reply:
@tobiasdiez  I have implemented the required changes in the recent commit.